src/Symfony/Component/DependencyInjection/Compiler/RegisterAutoconfigureAttributesPass.php

```diff
@@ -42,7 +43,8 @@ public function accept(Definition $definition): bool
 
     public function processClass(ContainerBuilder $container, \ReflectionClass $class): void
     {
-        foreach ($class->getAttributes(Autoconfigure::class, \ReflectionAttribute::IS_INSTANCEOF) as $attribute) {
+        $attributes = $class->getAttributes(Autoconfigure::class, \ReflectionAttribute::IS_INSTANCEOF) + $class->getAttributes(Lazy::class, \ReflectionAttribute::IS_INSTANCEOF);
```

Comment:
we should use array_merge instead of the `+` operator so that we don't loose attributes
can you please add a test case to cover this also?

Reply:
Will do!
Does this case also need a check to not use `#[Lazy]` when `#[Autoconfigure(lazy: true)]` is on, or should we just ignore the `#[Lazy]` in this case?

Reply:
I would do anything in case both attributes are there since we also allow two Autoconfigure attributes and we already have a rule to take many into account.

Reply:
Added a test to check adding both still works.
Just to be sure, what other cases did you want to check?